src/test/MyListImplementationTest.java

```diff
@@ -0,0 +1,140 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MyListImplementationTest {
+
+    @BeforeEach
+    void init(){
+    }
+
+    @Test
+    void testIsEmpty() {
+        MyListImplementation list =  new MyListImplementation<Integer>();
+        assertThat(list.isEmpty()).isEqualTo(true);
+    }
+
+    @Test
+    void testAdd() {
+        MyListImplementation list =  new MyListImplementation<Integer>();
+        list.add(1);
+        list.add(2);
+        assertThat(list.isEmpty()).isEqualTo(false);
+        assertThat(list.size()).isEqualTo(2);
+    }
+
+    @Test
+    void testContains() {
+        MyListImplementation list =  new MyListImplementation<Integer>();
+        list.add(1);
+        list.add(2);
+        assertThat(list.contains(1)).isEqualTo(true);

```

Comment:
Gerne in 2 Testfälle spalten. Einen Positiv-Test und einen Negativ-Test.

Das wären dann:
withListContainingValue_shouldReturnTrue
withListNotContainingValue_shouldReturnFalse

Grundsätzlich eine "fachlichkeit" pro Testfall!